manifests/r/RedGateSoftwareLtd/RedgateInstaller/14.0/RedGateSoftwareLtd.RedgateInstaller.installer.yaml

```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: RedGateSoftwareLtd.RedgateInstaller
+PackageVersion: 14.0
+Installers:
+- Architecture: x64
+  InstallerType: exe
+  InstallerUrl: https://download.red-gate.com/installers/SQLDeveloperBundle/2022-12-08/SQLDeveloperBundle.exe
+  InstallerSha256: 9158E0F4B9F086202291C77F31154C341033B9B6DBE279706D9E20EF0CDE9D80
+  InstallerSwitches:
+    Silent: /IAgreeToTheEULA /q

```

Comment:
This is a bit of a dangerous switch to add; Technically, this package should use the `Agreements` field if it requires EULA acceptance.